ezgui/src/drawing.rs

```diff
@@ -251,4 +254,20 @@ impl Prerender {
     pub(crate) fn request_redraw(&self) {
         self.inner.request_redraw()
     }
+
+    pub fn get_scale_factor(&self) -> f64 {
```

Comment:
`pub(crate)` for all of these new methods too

Reply:
done!